spec/system/passenger_brochure_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Passenger Brochure' do
+  let!(:staff_member) { create(:user, title: 'Admiral') }
+  let!(:non_staff_member) { create(:user) }
+  let!(:inactive_staff_member) { create(:user, active: false, title: 'Grand Poobah') }
+
+  before { visit brochure_passengers_path }
+
+  it 'shows users with titles' do
+    expect(page).to have_css('tr', text: /Admiral.*#{staff_member.name}/)
```

Comment:
Does this ignore all whitespace between 'Admiral' and the name?

Reply:
Yes, within the bounds of one `<tr>`. But, I think I want to achieve that slightly differently.